tests/unit/test_callbacks_writer_base.py

```
@@ -0,0 +1,8 @@
+import pytest
+
+from lighter.callbacks.writer.base import LighterBaseWriter
+
+
+def test_writer_initialization():
+    with pytest.raises(TypeError):
+        writer = LighterBaseWriter(path="test", writer="tensor")
```

Ruff (0.8.0)

Local variable `writer` is assigned to but never used

Remove assignment to unused variable `writer`

(F841)